pandas_transform/notebooks/dataframe_example.ipynb

```diff
@@ -0,0 +1,2363 @@
+       "      <td>1</td>\n",
+       "      <td>commercial</td>\n",
+       "      <td>inpatient</td>\n",
+       "      <td>0.3</td>\n",
+       "      <td>5.5</td>\n",
+       "      <td>1.2</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "         mrn first_name   last_name  gender  pan_day test_id  \\\n",
+       "27   5002114      azzak       tully    male       10   covid   \n",
+       "91   5011391        zei       umber  female       11   covid   \n",
+       "100  5000902       owen    seaworth    male       12   covid   \n",
+       "440  5005030     allard      rivers    male       18   covid   \n",
+       "472  5007953       anya  westerling  female       18   covid   \n",
+       "\n",
+       "          clinic_name    result  demo_group  age  drive_thru_ind  ct_value  \\\n",
+       "27   inpatient ward b  positive     patient  0.9               0     39.55   \n",
+       "91       s  care ntwk  positive     patient  0.1               0     39.83   \n",
+       "100    emergency dept  positive     patient  0.1               0     32.96   \n",
+       "440      clinical lab  positive  misc adult  0.0               1     16.30   \n",
+       "472    emergency dept  positive     patient  0.1               0     23.97   \n",
+       "\n",
+       "     orderset payor_group   patient_class  col_rec_tat  rec_ver_tat  \\\n",
+       "27          0  government     observation          0.6          2.7   \n",
+       "91          0  government      outpatient          5.2          4.8   \n",
+       "100         1  government       emergency          0.3          4.5   \n",
+       "440         0  unassigned  not applicable          2.4          5.6   \n",
+       "472         1  commercial       inpatient          0.3          5.5   \n",
+       "\n",
+       "     age_months  \n",
+       "27         10.8  \n",
+       "91          1.2  \n",
+       "100         1.2  \n",
+       "440         0.0  \n",
+       "472         1.2  "
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "infant_positive.loc[:,\"age_months\"] = infant_positive.loc[:,\"age\"]*12;\n",
+    "infant_positive.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d798fb7-3fe3-48d6-8ede-deb85ed16296",
+   "metadata": {},
+   "source": [
+    "**Note** You might be getting a warning, but even if the `SettingWithCopyWarning` warning appears, the DataFrame will still be modified the way you asked it to be.\n",
+    "\n",
+    "You can make changes to existing entires using the same method and simply using the key that already exists for that column. What if you wanted to reformat the gender column to us `M` and `F` instead of spelling out male and female? There are fancy ways to do this, but we already have tools to get the job done using conditions and the `.loc` method."
```

Comment:
* "entires" should be "entries" 
* "us" should be "use", in the line "...reformat the gender column to us `M` or `F`..."